src/hbbft_rbc.erl

```diff
@@ -224,13 +224,29 @@ check_completion(Data = #rbc_data{n=N, f=F}, H) ->
                             %% Figure2. Bullet6
                             %% check if we have enough readies and enough echoes
                             %% N-2F echoes and 2F + 1 readies
-                            case length(maps:get(H, Data#rbc_data.num_echoes, [])) >= M andalso length(maps:get(H, Data#rbc_data.num_readies, [])) >= (2*F + 1) of
-                                true ->
-                                    %% decode V. Done
-                                    {Data#rbc_data{state=done}, {result, Msg}};
+                            %% case length(maps:get(H, Data#rbc_data.num_echoes, [])) >= M andalso length(maps:get(H, Data#rbc_data.num_readies, [])) >= (2*F + 1) of
```

Comment:
So I'm guessing that this code used to do the exact same thing I'm doing below, the new code just follows the paper verbatim. If we don't need it, I'll revert to the old code.

Reply:
yeah, I thought as much. I'll  revert this since it's not exactly helpful